src/reporter.bif

```diff
@@ -22,7 +22,7 @@ module Reporter;
 ## .. zeek:see:: reporter_info
 function Reporter::info%(msg: string%): bool
 	%{
-	reporter->PushLocation(frame->GetCall()->GetLocationInfo());
+	reporter->PushLocation(frame->GetCall()->Original()->GetLocationInfo());
```

Comment:
Might it become error prone to remember or confusing to know when to do `Original()->GetLocationInfo()` vs. `GetLocationInfo()` ?  Could/should it work in a more generalized way such that asking an object for location info defaults to returning the original location (if there is one) ?

Reply:
Do you mean make `GetLocationInfo()` virtual, and override it for those classes (`Expr`, `Stmt`) that have the notion of `Original()`?  I agree that that's cleaner.  It might require adding some logic to get the non-Original location in some circumstances, but that's likely better isolated.